HelpSource/Classes/Object.schelp

```diff
@@ -48,20 +48,34 @@ Create a new instance. The creation of new instances of any Object actually happ
 
 method::newCopyArgs
 
-Creates a new instance and copies the arguments to the instance variables in the order that the variables were defined. If the superclass's code::new:: method requires arguments, the first arguments passed to code::newCopyArgs:: will be passed on to that method, and the following arguments will be copied to this class's instance variables. Class variables are ignored.
+Creates a new instance and copies the arguments to the instance variables in the order that the variables were defined.
+
+Importantly, super class constructors and NOT called, but the first values will be copied.
+Think of instance variables as entries in an array,
+the super class's instance variables are at the beginning and the child class's at the end,
+this method only copies the arguments into that array.
 
 code::
 MyClass {
 	var a, b, c;
+    // Will copy arg1, arg2, arg3 to variables a, b, c.
+	*new { |arg1, arg2, arg3| ^super.newCopyArgs(arg1, arg2, arg3) }
+	value { ^[a, b, c] }
+}
+MyClass(1, 2, 3).() == [1, 2, 3]
```

Comment:
In this case perhaps it's best to just call `.value` instead of `.()`, to be explicit.